lib/cli/commands-schema/no-service.js

```diff
@@ -39,6 +39,12 @@ commands.set('', {
       type: 'string',
       shortcut: 'f',
     },
+    'verbose': {
+      usage:
+        'Include more dev mode information including the body of requests and responses as well as input and output of incoming spans.',
+      type: 'string',
+      shortcut: 'v',
+    },
```

Comment:
This option is already defined in common options set: https://github.com/serverless/serverless/blob/a2d777ae8303fb96e35caf3754435e3742a7c286/lib/cli/commands-schema/common-options/global.js#L6

It serves our logging, so with this flag we produce more verbose output. I think you can just take advantage of it in dev mode as well

Reply:
Ahh okay nice thank you for pointing this out 👍